jekyll/_docs/ios-builds-on-os-x.md

```diff
@@ -136,9 +136,9 @@ While CircleCI's inferred commands will handle many common testing patterns, you
 also have a lot of flexibility to customize what happens in your build.
 
 ### Build Commands
-CircleCI runs tests from the command line with the [`xctool`](https://github.com/facebook/xctool)
-command by default. We have found that `xctool` is more stable when testing with
-the iOS simulator than using `xcodebuild` directly.
+CircleCI runs tests from the command line with the [`xcodebuild`](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/xcodebuild.1.html)
+command by default. This is a tool developed by Apple, and we found it to be the most
```

Comment:
"... and we *find* it  to be the"  as opposed to the past-tense found.  but honestly ~~